.hlint.yaml

```
@@ -9,11 +9,18 @@
   - {name: Data.Map, as: Map, message: "Use complete name for qualified import of Map"}
   - {name: Data.Map.Strict, as: Map, message: "Use complete name for qualified import of Strict Map"}
   - {name: Data.Text, as: Text, message: "Use complete name for qualified import of Text"}
+  # Don't allow unsafe IO
+  - {name: System.Unsafe.IO, within: []}
```

https://hackage.haskell.org/package/base-4.15.0.0/docs/System-IO-Unsafe.html
```suggestion
  - {name: System.IO.Unsafe, within: []}
```